var/spack/repos/builtin/packages/heffte/package.py

```diff
@@ -26,10 +26,16 @@ class Heffte(CMakePackage, CudaPackage, ROCmPackage):
     version("2.3.0", sha256="27c0a8da8f7bc91c8715ecb640721ab7e0454e22f6e3f521fe5acc45c28d60a9")
     version("2.2.0", sha256="aff4f5111d3d05b269a1378bb201271c40b39e9c960c05c4ef247a31a039be58")
     version("2.1.0", sha256="527a3e21115231715a0342afdfaf6a8878d2dd0f02f03c92b53692340fd940b9")
```

Comment:
I am not getting these sha256 for versions 1.0, 2.0.0,  2.1.0, 2.2.0, or 2.3.0 with the new URL using this branch.  You could add `url_for_version()` or a `url` argument to each version statement to fix this if you don't want to change the `sha256`.

Reply:
FYI. https://spack.readthedocs.io/en/latest/packaging_guide.html#version-urls.

Reply:
We moved the repo, which messed up the sha checksums. I've updated those.